test/e2e/runner/start.go

```diff
@@ -66,8 +69,22 @@ func Start(testnet *e2e.Testnet) error {
 		}
 	}
 
-	// Start up remaining nodes
 	for _, node := range nodeQueue {
+		// if we're starting a node in the "future" relative
+		// to where the network currently is, we should make
+		// sure the network has gotten to that point first.
+		if node.StartAt > initialHieght {
```

Comment:
```suggestion
		if node.StartAt > initialHeight {
```

Reply:
Please also add a comment as to what is going on here. Perhaps (assuming this is right)

```go
// Ensure the target initial height is the maximum among the starting heights
// of all the nodes currently in the queue, if there are any. This averts the case
// where we might attempt to "catch up" to a height that has already passed.
```

Reply:
Your example comment is, I think, exactly backwards, so I amended my comment. and moved it around to be a bit more clear.